### DISCUSSION TITLE
checkpoints

### DISCUSSION THREAD

**Discussion by KoopReynders** [fdnd/docs.fdnd.nl#99] [Onderwijs] [answered]
Het stuk over _checkpoints_, _feedback schrijven_ en _Verhouding checkpoints assessments_ herschrijven naar de nieuwe werkwijze: 

Agile leren: Sprintplanning > Opdrachten en opdrachtgever > workshops, talks > feedback in issues (GH workflow) > code/design review > Retrospect

**Feedback**
We doen geen checkpoints meer:
Inhoudelijke feedback op code, design, schetsen, samenwerken komt in issues
Feedback op criteria in Portflow na een retrospect door docenten en (peer) feedback op gedragsindicatoren

**Comment by ju5tu5** [comment, marked as answer]
Mooie iteratie op basis van de HEAL voor het zomerreces. Issue gemaakt, docs is inmiddels aangepast.
